src/excmds.ts

```diff
@@ -3316,25 +3316,41 @@ export function autocmd(event: string, url: string, ...excmd: string[]) {
     config.set("autocmds", event, url, excmd.join(" "))
 }
 
-/** Automatically open a domain and all its subdomains in a specified container.
+/** Automatically open matching URLs in a specified container.
+ *
+ *  This function accepts a `-u` flag to treat the pattern as a URL rather than a domain.
+ *  For example: `autocontain -u ^https?://[^/]*youtube\.com/ google` is equivalent to `autocontain youtube\.com google`
```

Comment:
AFAICT this does the exact opposite of what it says: `-u` makes it treat the thing as a regex. `-r` for regex or maybe `-e` for extended mode makes more sense?

Reply:
As discussed on IRC the distinction here is "domain" vs "url", and the argument is always a regex. The lack of `-u` just means we wrap it in the `https?://[^/]<input>/` pattern so it doesn't match arbitrary URLs. Part of the issue here is that I changed the wording before adding the compatibility mode in, so I've reverted some of that to make the docs change more minimal which should help clear it up?